lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1780,57 +1655,72 @@ private void createTables(String indexName, List<IndexingRules> indexingRules) t
         props.put(OBJ_PROV_COMMIT_HASH, keyword);
         props.put(OBJ_MD5, keyword);
 
-        tmp = new LinkedHashMap<>();
-        tmp.put("type", "date");
+        tmp = ImmutableMap.of("type", "date");
         props.put("timestamp", tmp);
 
-        tmp = new LinkedHashMap<>();
-        tmp.put("type", "keyword");
+        tmp = ImmutableMap.of("type", "keyword");
```

Comment:
There's already a keyword var for this above, just do props.put("prefix", keyword)